tests/manual-test-cases/Group18-VIC-UI/kickoff-tests.robot

```diff
@@ -38,8 +38,8 @@ Prepare Testbed
     Check Working Dir
     Check Drone
     Check Govc
-    Install VIC Product OVA  6.0u2  ${BUILD_3634791_IP}  10.192.42.90  datastore1 (3)
-    Install VIC Product OVA  6.5d  ${BUILD_5318154_IP}  10.160.75.194  datastore1 (1)
+    Install VIC Product OVA  6.0u2  ${BUILD_3634791_IP}  %{OVA_ESX_IP_VC60U2}  %{OVA_ESX_DATASTORE_VC60U2}
+    Install VIC Product OVA  6.5d  ${BUILD_5318154_IP}  %{OVA_ESX_IP_VC65D}  %{OVA_ESX_DATASTORE_VC65D}
```

Comment:
On the download page for vsphere I see U1d listed as the latest vsphere. https://my.vmware.com/en/web/vmware/info/slug/datacenter_cloud_infrastructure/vmware_vsphere/6_5

Is that the version we are testing? If not when can we switch to testing this version? We may be testing an obsolete version.

Reply:
No, we are testing against 6.5d not 6.5u1d. The plugin works on 6.5u1 but haven't yet been tested on 6.5u1d. I plan to set up a vCenter 6.5u1 (or 6.5u1d) instance soon on our Dell server and replace the obsolete 6.5d but doing so itself is a major work that might take more effort and time than you think. If I go ahead and do it I expect at least one day of down time for our nightly tests